src/int/expand.rs

```diff
@@ -0,0 +1,17 @@
+use crate::{ConstantTimeSelect, Int, Uint};
+
+impl<const LIMBS: usize> Int<LIMBS> {
+    /// Expand the representation of `self` to an `Int<T>`.
+    /// Assumes `T >= LIMBS`; panics otherwise.
+    #[inline(always)]
+    pub fn expand<const T: usize>(&self) -> Int<T> {
+        assert!(T >= LIMBS);
```

Comment:
make it debug_assert (the same as Uint does)

Reply:
was removed.